docs/documentation/test_matrix.md

```diff
@@ -1,14 +1,34 @@
 # Hardware test matrix
 
-Devices and configurations TrenchBoot has been tested on.
-
-| Tested devices                                                               | TPM family | Result | Notes |
-|:----------------------------------------------------------------------------:|:----------:|:------:|:-----:|
-| Intel Tiger Lake client                                                      |  TPM 2.0   |  PASS  | UEFI firmware |
-| Intel Kaby Lake server                                                       |  TPM 2.0   |  PASS  | UEFI firmware |
-| Intel Skylake sever                                                          |  TPM 2.0   |  PASS  | UEFI firmware |
-| PC Engines APU2 platform series <br>(AMD family 16h models 30h-3fh embedded) |  TPM 2.0   |  PASS  | coreboot firmware |
-| PC Engines APU2 platform series <br>(AMD family 16h models 30h-3fh embedded) |  TPM 1.2   |  PASS  | coreboot firmware |
-| Asus KGPE-D16 <br>(AMD Opteron family 15h models 00h-0fh server)             |  TPM 2.0   |  FAIL  | coreboot firmware TPM issue |
-| Supermicro M11SDV-8CT <br>(AMD EPYC 3000 Snowy Owl server)                   |  TPM 2.0   |  PASS  | CSM legacy boot |
-| Supermicro M11SDV-8CT <br>(AMD EPYC 3000 Snowy Owl server)                   |  TPM 2.0   |  FAIL  | UEFI boot |
\ No newline at end of file
+Devices and configurations on which TrenchBoot is known to work (availability
```

Comment:
I would also add [HP Thin Client t630](https://support.hp.com/us-en/document/c05240287) to this list. Except problem with starting original ISO in legacy mode (similar to https://github.com/QubesOS/qubes-issues/issues/8462#issuecomment-1697323478, but it only worked for me after i also changed partition type from protective MBR to Linux, in addition to marking it active), everything works out of the box. I'm not sure if it can be bought as new, but [BIOS was updated two months ago](https://support.hp.com/us-en/drivers/hp-t630-thin-client/10522151) so at least it is still supported by vendor.

![image](https://github.com/TrenchBoot/TrenchBoot.github.io/assets/40995177/520b10f8-5007-4fd2-be4d-10d39dd9e208)

Reply:
Added in ea216986222c99ad18fbbcd0f74342fded899008 linking to your comment.